packages/augur-ui/src/modules/create-market/set-categories.ts

```diff
@@ -8,8 +8,8 @@ export const setCategories: SortedGroup[] = [
     label: 'Sports',
     subGroup: [
       {
-        value: 'Soccer',
-        label: 'Soccer',
+        value: 'Football (Soccer)',
+        label: 'Football (Soccer)',
```

Comment:
I'm not sure if this breaks anything. Should this value remain `Soccer` or can it be updated?